modules/ballerina-core/src/main/java/org/ballerinalang/model/values/BTypeValue.java

```diff
@@ -46,7 +46,7 @@ public BType value() {
 
     @Override
     public String stringValue() {
-        return null;
+        return typeValue.getName() + ".type";
```

Comment:
I see several issues in this approach.

1) If I want to print the type of a struct in a package, then this will only print the struct name. The output has to be in the form of 'pkg-name:struct-name'.
2) Remove .type suffix. Just print the type name and the package name.